modules/installation-requirements-user-infra.adoc

```diff
@@ -155,7 +155,7 @@ ifndef::ibm-power[|4]
 ifndef::openshift-origin[]
 |Compute
 ifdef::ibm-z,ibm-power[|{op-system}]
-ifndef::ibm-z,ibm-power[|{op-system} or RHEL 7.8 - 7.9]
+ifndef::ibm-z,ibm-power[|{op-system}, RHEL 7.9, or RHEL 8]
```

Comment:
@marrusl - I'm not clear whether this should be `RHEL 8` or `RHEL 8.4` specifically for compute machines. Please clarify.

Reply:
RHEL 8 Server support (non-RHCOS) won't be in OCP 4.8. Only support for RHEL 7.9 Server and RHCOS.

Reply:
Thanks for clarifying, @katherinedube. I have removed the RHEL 8 reference.